3634/1/gems.rb

```diff
@@ -0,0 +1,27 @@
+class Gems
+  attr_reader :names
+
+  def initialize(gems)
+    @names = gems(gems)
+    @paths = []
```

Comment:
no reason to create this instance variable here since you can create this with .map in the `paths` method